zk_toolbox/crates/zk_supervisor/src/commands/test/revert.rs

```diff
@@ -22,12 +24,27 @@ pub fn run(shell: &Shell, args: RevertArgs) -> anyhow::Result<()> {
 
 fn install_and_build_dependencies(
     shell: &Shell,
+    args: &RevertArgs,
     ecosystem_config: &EcosystemConfig,
 ) -> anyhow::Result<()> {
     let _dir_guard = shell.push_dir(&ecosystem_config.link_to_code);
-    let spinner = Spinner::new("Installing and building dependencies...");
+    let spinner = Spinner::new(MSG_REVERT_TEST_INSTALLING_DEPENDENCIES);
     Cmd::new(cmd!(shell, "yarn install")).run()?;
     Cmd::new(cmd!(shell, "yarn utils build")).run()?;
+
+    // Build always zk_server and block_reverter binaries
+    Server::new(None, ecosystem_config.link_to_code.clone()).build(shell)?;
+    Cmd::new(cmd!(shell, "cargo build --release --bin block_reverter")).run()?;
```

Comment:
why do we need it? it suppose to work even without prebuilding

Reply:
Double-checking this with Agus (it's part of his initial commit).

Reply:
Removed in https://github.com/matter-labs/zksync-era/pull/2408/commits/aac7164cacefbd9cef86486761b45c1e37ab7ed5